stumpy/core.py

```diff
@@ -1258,9 +1265,22 @@ def mass_absolute(Q, T, T_subseq_isfinite=None, p=2.0):
     if np.any(~np.isfinite(Q)):
         distance_profile[:] = np.inf
     else:
+        if query_idx is not None:
+            query_idx = int(query_idx)
+            if not np.allclose(Q, T[query_idx : query_idx + m]):
+                msg = (
```

Comment:
Same as my other comment, we will get wrong warning if the provided input `T` is the preprocessed version of `T` and `T_subseq_isfinite` is provided by the user.

Reply:
See comment below